tests/test_compose.py

```diff
@@ -247,20 +247,38 @@ def test_service_parser_volumes(self):
 
         binds_config = {
             'Binds': {
-                '/tmp:/tmp'
+                'c:\\tmp:/tmp'
             }
         }
 
         volumes_list = [
             {
                 'target': '/tmp',
                 'type': 'bind',
-                'source': '/tmp'
+                'source': 'c:\\tmp'
             }
         ]
 
         assert volumes_list == iotedgehubdev.compose_parser.service_parser_volumes(binds_config)
 
+        invalid_binds_config = {
+            'Binds': {
+                'c:\\tmp:/tmp:r'
```

Comment:
is case 'c:/tmp:/tmp:r' valid?

Reply:
No. If there is a 3rd part, it must be `ro` to indicate read-only.